.github/workflows/release.yml

```diff
@@ -0,0 +1,84 @@
+name: Build & publish binary node
+
+on:
+  release:
+    types: [published]
+
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - windows-latest
+        host:
+          - x64
+        target:
+          - x64
+        node:
+          # - 10 # Issue with gyp build can build with this version.
+          - 12
+          - 14
+          - 16
+          - 18
+        include:
+          - os: windows-latest
+            node: 18
+            host: x86
+            target: x86
+
+    name: ${{ matrix.os }} (node=${{ matrix.node }}, host=${{ matrix.host }}, target=${{ matrix.target }})
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1.1
+        with:
+          msbuild-architecture: ${{ matrix.target }}
+
+      - name: Setup node (node=${{ matrix.node }}, host=${{ matrix.host }})
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          architecture: ${{ matrix.host }}
+      - name: Install dependencies
+        run: yarn install --ignore-scripts
```

Comment:
I prefer to use `npm`.

Reply:
It's more faster then npm. I base the release script on the [node-sqlite3 project](https://github.com/TryGhost/node-sqlite3/blob/master/.github/workflows/ci.yml). This part has no impact on the viability of the project.

Reply:
> It's more faster then npm.

The CI step will be slow anyway. I'm the one stuck maintaining all this code. I don't want to depend on Yarn.